File: scripts/seed_vuln.py
"""Exists only on the throwaway seed/codeql-2026-09-05 branch, to prove CodeQL's
py/command-line-injection query fires against the production analysis. Never merged."""

import subprocess
import sys


def run(name):
    subprocess.run(f"echo {name}", shell=True)


if __name__ == "__main__":
    run(sys.argv[1])

File: frontend/src/seed-vuln.ts
// Exists only on the throwaway seed/codeql-2026-09-05 branch, to prove CodeQL's js/xss query
// fires against the production analysis. Never merged.
const target = document.getElementById("output");
if (target !== null) {
  target.innerHTML = location.hash.substring(1);
}
